src/map/alghom.jl

```diff
@@ -0,0 +1,142 @@
+export Algebra_Homomorphism, codomain, compose, domain, kernel, preimage
```

Comment:
No underscores in CAMLCase names. Needs changing throughout.

Reply:
I will change this. 
My problem at the moment is, that the abstract type is also called "AlgebraHomomorphism". I would change the abstract type to "AlgebraMorphism", if this does not interfere with any convention.